meta-cli/src/typegraph/mod.rs

```diff
@@ -14,30 +17,24 @@ use walkdir::{DirEntry, WalkDir};
 
 use crate::config::Config;
 use crate::utils::ensure_venv;
-use typescript::parser::{transform_module, transform_script};
+use postprocess::postprocess;
 
 pub type LoaderResult = HashMap<String, Result<Vec<Typegraph>>>;
 
 #[derive(Clone)]
 pub struct TypegraphLoader<'a> {
     skip_deno_modules: bool,
     ignore_unknown_file_types: bool,
+    deploy: bool, // target is deployment
```

Comment:
I would rather put an enum `target = Target.deployment or Target.Serialize` to explicite this parameter.